db/migrate/20210716220754_create_sms_messages.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSmsMessages < ActiveRecord::Migration[5.2]
+  def change
+    create_table :sms_messages do |t|
+      t.string :mobile_number
+      t.string :message
```

Comment:
We only have the two parameters on our side so this will work well for our requests!

Reply:
Great! Glad we had a chance to work on this component of the project.